ena-submission/src/ena_deposition/create_assembly.py

```diff
@@ -438,10 +455,10 @@ def assembly_table_create(
             db_config, sample_data_in_submission_table[0]
         )
 
-        if is_revision(db_config, seq_key) and not can_be_revised(
-            config, db_config, sample_data_in_submission_table[0]
-        ):
-            continue
+        if is_revision(db_config, seq_key):
+            logger.debug(f"Entry {row['accession']} is a revision, checking if it can be revised")
+            if not can_be_revised(config, db_config, sample_data_in_submission_table[0]):
+                continue
```

Comment:
Log that it can't be revised when it can't be revised?

Reply:
there are logs in the `can_be_revised` function if we hit that case - they are more detailed